### PR TITLE
feat(backend): add an id to requests

### PR DESCRIPTION
**Pull request by virtuacoplenny**

### CODE REVIEW

File: spot-client/setupTests.js [hunk 1]
```diff
@@ -6,3 +7,7 @@ Enzyme.configure({ adapter: new EnzymeAdapter() });
 jest.mock('./src/common/logger');
 
 global.fetch = require('jest-fetch-mock');
+
+global.crypto = {
+    getRandomValues: buffer => crypto.randomFillSync(buffer)
```
Comment (paweldomas) [inline spot-client/setupTests.js]: why this change ?

Comment (paweldomas) [reply spot-client/setupTests.js]: just not sure how is it related to adding `request-id` header

Comment (virtuacoplenny) [reply spot-client/setupTests.js]: guid creation. I could mock out the guid method itself, if that's preferred.
